packages/ra-supabase-core/src/dataProvider.ts

```diff
@@ -40,7 +41,18 @@ export const supabaseDataProvider = ({
         schema,
         ...rest,
     };
-    return postgrestRestProvider(config);
+    return {
+        supabaseClient: (url: string, options?: any) =>
+            httpClient(`${config.apiUrl}/${url}`, options),
```

Comment:
It is not, but I initially wanted to add the getSchema method directly in the demo app, only to realize I was missing the ability to call the API. This will prove very useful for custom endpoints.